src/plugins/controls/public/options_list/components/options_list.scss

```diff
@@ -47,19 +51,16 @@
 .optionsList__popoverOverride {
   @include euiBottomShadowMedium;
   filter: none; // overwrite the default popover shadow
-  transform: translateY(-$euiSizeS) translateX(0); // prevent "slide in" animation on open/close
```

Comment:
@Heenawter This was the style that shifted the options list popovers up 8px. Is this okay to remove?

Reply:
Ahhh! Good catch - I probably forgot to remove that as part of https://github.com/elastic/kibana/pull/165397 🤦 You're totally right - that was only necessary when we were using a normal `EuiPopover`; as soon as we switched to `EuiInputPopover` in that attached PR, this became redundant.